sixtyfps_compiler/generator/cpp.rs

```diff
@@ -10,6 +10,8 @@ LICENSE END */
 /*! module for the C++ code generator
 */
 
+// cSpell:ignore cstdlib cmath constexpr nullptr decltype intptr uintptr
```

Comment:
Won’t those also show up in C++ sources? Just curious

Reply:
Nope, the keywords of the language in use are ignored by cspell.

Reply:
Ah, great, that's good. Thanks for the explanation :)